ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+> ℹ Please fill out this template when filing an issue.
+> All lines beginning with a quotation mark (`>`),
+> instruct you with what info we expect.  
+> Please remove this opening paragraph.
+
+### Goals
+
+> ℹ Please replace this with what you tried to achieve.
+
+### Expected Results
+
+> ℹ Please replace this with what you expected to happen.
+
+### Actual Results
+
+> ℹ Please replace this with what actually happened.  
+> e.g. the stack trace of a crash
```

Comment:
I see your point, but I think it would be more important that stack traces are formatted as source code. As long as that is given, I think it's still relatively easy to keep an overview, especially given that the headings visually stand out.